packages/cli/src/modules/insights/database/repositories/insights-by-period.repository.ts

```diff
@@ -21,6 +21,32 @@ const summaryParser = z
 	})
 	.array();
 
+const aggregatedInsightsByWorkflowParser = z
+	.object({
+		workflowId: z.string(),
+		workflowName: z.string().optional(),
+		projectId: z.string().optional(),
+		projectName: z.string().optional(),
+		total: z.union([z.number(), z.string()]),
+		succeeded: z.union([z.number(), z.string()]),
+		failed: z.union([z.number(), z.string()]),
+		failureRate: z.union([z.number(), z.string()]),
+		runTime: z.union([z.number(), z.string()]),
+		averageRunTime: z.union([z.number(), z.string()]),
+		timeSaved: z.union([z.number(), z.string()]),
```

Comment:
Why are they number or string? Is this because some driver returns strings and others numbers/dates? If that's the case, would it make sense to transform them?

Reply:
I see you're doing this in the service. We can move it into the zod schema and simplify the service. WDYT?

Reply:
@despairblue good idea ! and yes it's because some drivers return string for number (for bigint or high precision decimal I think)